FLEDGE.md

```diff
@@ -304,32 +304,46 @@ Buyers may want to make on-device decisions that take into account real-time dat
 
     https://www.kv-server.example/getvalues?hostname=publisher.com&keys=key1,key2
 
-The base URL `https://www.kv-server.example/getvalues` comes from the interest group's `trustedBiddingSignalsUrl`, the hostname of the top-level webpage where the ad will appear `publisher.com` is provided by the browser, and `keys` is a list of `trustedBiddingSignalsKeys` strings, perhaps coalesced (for efficiency) across any number of interest groups that share a `trustedBiddingSignalsUrl`.  The response from the server should be a JSON object whose keys are key1, key2, etc., and whose values will be made available to the buyer's bidding functions (un-coalesced).
+The base URL `https://www.kv-server.example/getvalues` comes from the interest group's `trustedBiddingSignalsUrl`, the hostname of the top-level webpage where the ad will appear `publisher.com` is provided by the browser, and `keys` is a list of `trustedBiddingSignalsKeys` strings.  The requests may be coalesced (for efficiency) across any number of interest groups that share a `trustedBiddingSignalsUrl` (which means they also share an owner).
+
+The response from the server should be a JSON object of the form:
+
+```
+{ 'keys': {
+      'key1': arbitrary_json,
+      'key2': arbitrary_json,
+      ...}
+}
+```
+
+and the server must include the HTTP response header "X-fledge-bidding-signals-format-version: 2".  If the server does not include the header, the response will assumed to be an in older format, where the response is only the contents of the `keys` dictionary.
```

Comment:
```suggestion
and the server must include the HTTP response header `X-fledge-bidding-signals-format-version: 2`.  If the server does not include the header, the response will assumed to be an in older format, where the response is only the contents of the `keys` dictionary.
```

Reply:
I'm not sure about this.  Should back ticks be used instead of " eveywhere?  We use back ticks for variable names and code blocks, but I'm not sure about quoting individual headers.

Reply:
I like back ticks for this because they there is no quote or double-quote mark in the explainer so folks don't do silly things like put quotes into their HTTP headers.  This is obviously a small change that I don't feel strongly about so feel free to reject this change.